src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ComboBox.ComboBoxUiaTextProviderTests.cs

```diff
@@ -0,0 +1,1161 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Drawing;
+using System.Windows.Forms.Automation;
+using Xunit;
+using static Interop;
+using static Interop.User32;
+
+namespace System.Windows.Forms.Tests
+{
+    public class ComboBox_ComboBoxUiaTextProviderTests : IClassFixture<ThreadExceptionFixture>
+    {
+        [WinFormsTheory]
+        [InlineData(ComboBoxStyle.DropDown)]
+        [InlineData(ComboBoxStyle.Simple)]
+        public void ComboBoxUiaTextProvider_Ctor_DoesntCreateControlHandle(ComboBoxStyle dropDownStyle)
+        {
+            using (new NoAssertContext())
```

Comment:
In most cases you can do this:
```suggestion
            using NoAssertContext context = new();
```
It the same, just less braces and indentation.